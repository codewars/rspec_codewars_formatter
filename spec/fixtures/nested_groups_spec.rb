RSpec.describe "Group 1" do
  describe "Group 1.1" do
    it "passes" do
      expect(true).to eq(true)
    end
  end
end
