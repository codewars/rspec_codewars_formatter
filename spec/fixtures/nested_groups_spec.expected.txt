
<DESCRIBE::>Group 1

<DESCRIBE::>Group 1.1

<IT::>passes

<PASSED::>Test Passed

<COMPLETEDIN::>0.6725

<COMPLETEDIN::>1.1803

<COMPLETEDIN::>1.3343
